compose.yaml

```diff
@@ -0,0 +1,30 @@
+services:
+  db:
+    image: postgres:17.2-alpine3.21
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - "5433:5432"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+  app:
+    build: .
+    command: >
+      sh -c "python manage.py wait_for_db &&
+            python manage.py migrate &&
+            python manage.py runserver 0.0.0.0:8000"
```

Comment:
Consider using a process manager like Gunicorn for running the Django application in production instead of the built-in Django development server, which is not suitable for production use.